code/mod_table/mod_table.cpp

```diff
@@ -1443,6 +1444,7 @@ void mod_table_reset()
 	Default_detail_level = 3; // "very high" seems a reasonable default in 2012 -zookeeper
 	Full_color_head_anis = false;
 	Dont_automatically_select_turret_when_targeting_ship = false;
+	Automatically_select_subsystem_underreticle_when_targeting_ship = false;
```

Comment:
Well, you introduce this bool to toggle your feature, but to actually make use of it, you need to actually parse a value from the table and populate the bool.
It should probably be somewhere towards the end of the #Other settings group of settings.

Reply:
#5145 should provide a simple roadmap to working with mod_table.cpp to add the necessary boolean table flag.

Reply:
I was thinking HUD settings?  Similar to the Dont_automatically_select_turret_when_targeting_ship, which has some code in the same area.